modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,104 @@
 
-[id="stable-4-channel_{context}"]
-== stable-4 channel
-Releases are added to the `stable-4` channel after passing all tests.
+[NOTE]
+Because the time required to obtain a statistically significant number of data points varies based on many factors we offer no Service LeveL Objective (SLO) for the delay duration between fast and stable channels. Please see link:strategies_{context}[Fast and stable channel use and strategies] for more information.
 
-You can use the `stable-4` channel to upgrade from a previous minor version of {product-title}.
-endif::openshift-origin[]
 
-ifndef::openshift-origin[]
+Newly installed clusters default to utilizing stable channels.
 
 [id="eus-4y-channel_{context}"]
 == eus-4.y channel
 
-In addition to the stable channel, all even-numbered minor versions of {product-title} offer an link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support] (EUS). These EUS versions extend the Full and Maintenance support phases for customers with Standard and Premium Subscriptions to 18 months.
-
-Although there is no difference between `stable-4.y` and `eus-4.y` channels until {product-title} 4.y transitions to the EUS phase, you can switch to the `eus-4.y` channel as soon as it becomes available.
-
-When upgrades to the next EUS channel are offered, you can switch to the next EUS channel and upgrade until you have reached the next EUS version.
-
-This upgrade process does not apply for the `eus-4.6` channel.
+In addition to the stable channel, all even-numbered minor versions of {product-title} offer link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support] (EUS). Releases which are promoted to the stable channel are also simultaneously promoted to the EUS channels. The primary purpose of the EUS channels is to serve as a convenience for clusters performing an EUS-to-EUS update.
 
 [NOTE]
 ====
 Both standard and non-EUS subscribers can access all EUS repositories and necessary RPMs (`rhel-*-eus-rpms`) to be able to support critical purposes such as debugging and building drivers.
 ====
 
+[id="candidate-version-channel_{context}"]
+== candidate-{product-version} channel
+
+The `candidate-{product-version}` channel offers unsupported early access to releases as soon as they're built. Releases present only in candidate channels
+may not contain the full feature set of eventual GA releases or features may be removed prior to GA. Additionally, these releases have not been subject to full
+Red Hat Quality Assurance and may not offer update paths to later GA releases. Given these caveats the candidate channel is only suitable for testing purposes
+where destroying and recreating a cluster is acceptable.
```

Comment:
@julienlim here's the updated section on candidate channels
- move unsupported'ness front and center
- drops the assertion that items in candidate are always feature complete and grants us ability to drop features